BuildAndTest.proj

```diff
@@ -1,7 +1,7 @@
 <Project
   xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
   DefaultTargets="BuildAndTest"
-  ToolsVersion="12.0">
+ >
```

Comment:
So, you were correct :)

Not having any tools version causes VS to treat it as a .NET 1.0 project. Defining it as 4.0 is the minimum we need, but defining it to be equivalent to the VisualStudioVersion (currently 14.0, which is what we still use for the signed build machines currently) seems to be the best option (as it causes it to target the correct 'sub-toolset').